pkgs/applications/graphics/oculante/default.nix

```diff
@@ -11,12 +11,21 @@
 , libXrandr
 , libXi
 , libGL
+, libxkbcommon
+, wayland
 , stdenv
 , gtk3
 , darwin
 , perl
 }:
 
+let
+  rpathLibs = [
```

Comment:
can we inline this? since it's not used anywhere else, and `postFixup` only applies when `stdenv.isLinux`